PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -4,70 +4,79 @@ This document provides information on how to get your repository set up to take
 
 ## Account Setup
 
-First you will need a Router account, and to configure it for SWORDv2 integration.  See your account page in the 
-router for more information.  You need to provide the following:
+Firstly, you will need to contact pubrouter@jisc.ac.uk to create an account. For that, you must provide the matching parameters [template](https://pubrouter.jisc.ac.uk/static/csvtemplate.csv) and the SWORDv2 credentials:
 
-1. A username and password for a user in your repository who has the rights to create content via SWORDv2
-2. A collection URL into which the system will deposit the content
-3. Your preferred packaging format for zip files being deposited.  By default (and the only option in this first version) this will be http://purl.org/net/sword/package/SimpleZip
+* A *username* and *password* for a user in your repository who has the rights to create content via SWORDv2.
+   * For Eprints deposits into *Manage Deposits* a User role is enough.
+   * For Eprints deposits into *Review* queue an Editor or Admin role is required.
```

Comment:
For Eprints deposits into **the** Review queue, an Editor or Admin role is required.